.github/workflows-source/release-testing.yml

```diff
@@ -0,0 +1,99 @@
+  CI_JOB_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+  CI_PIPELINE_SOURCE: ${{ github.event_name }}
+  CI_PROJECT_DIR: ${{ github.workspace }}
+  CI_MERGE_REQUEST_TARGET_BRANCH_NAME: ${{ github.event.pull_request.base.ref }}
+  ROOT_PIPELINE_ID: ${{ github.run_id }}
+  BAZEL_STARTUP_ARGS: "--output_base=/var/tmp/bazel-output/"
+  RUSTFLAGS: "--remap-path-prefix=${CI_PROJECT_DIR}=/ic"
+  AWS_SHARED_CREDENTIALS_CONTENT: ${{ secrets.AWS_SHARED_CREDENTIALS_FILE }}
+  DOCKER_HUB_USER: ${{ secrets.DOCKER_HUB_USER }}
+  DOCKER_HUB_PASSWORD_RO: ${{ secrets.DOCKER_HUB_PASSWORD_RO }}
+  BUILDEVENT_APIKEY: ${{ secrets.HONEYCOMB_API_TOKEN }}
+  BUILDEVENT_DATASET: "github-ci-dfinity"
+
+anchors:
+  image: &image
+    image: ghcr.io/dfinity/ic-build@sha256:5bd0f059ad6e21966e9c644516b6ecd52d5ed44f1b18a76b91b59740a9d639a2
+  dind-large-setup: &dind-large-setup
+    runs-on:
+      group: dind-large
+    container:
+      <<: *image
+    timeout-minutes: 180 # 3 hours
+    if: ${{ vars.RUN_CI == 'true' }}
+  checkout: &checkout
+    name: Checkout
+    uses: actions/checkout@v4
+  before-script: &before-script
+    name: Before script
+    id: before-script
+    shell: bash
+    run: ./gitlab-ci/src/ci-scripts/before-script.sh
+
+jobs:
+  bazel-system-test-nightly:
+    name: Bazel System Test Nightly
+    <<: *dind-large-setup
+    steps:
+      - <<: *checkout
+      - <<: *before-script
+      - name: Run Bazel System Test Nightly
+        id: bazel-test-all
+        uses: ./.github/actions/bazel-test-all/
+        with:
+          BAZEL_COMMAND: "test"
+          BAZEL_TARGETS: "//rs/tests/..."
+          BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
+          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=system_test_nightly --deleted_packages=gitlab-ci/src/gitlab_config"
+          HONEYCOMB_API_TOKEN: ${{ secrets.HONEYCOMB_API_TOKEN }}
+
+  bazel-system-test-staging:
+    name: Bazel System Test Staging
+    continue-on-error: True
+    <<: *dind-large-setup
+    steps:
+      - <<: *checkout
+      - <<: *before-script
+      - name: Run Bazel System Test Staging
+        id: bazel-test-all
+        uses: ./.github/actions/bazel-test-all/
+        with:
+          BAZEL_COMMAND: "test"
+          BAZEL_TARGETS: "//rs/tests/..."
+          BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
+          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=system_test_staging --deleted_packages=gitlab-ci/src/gitlab_config"
+          HONEYCOMB_API_TOKEN: ${{ secrets.HONEYCOMB_API_TOKEN }}
+
+  bazel-system-test-hotfix:
+    name: Bazel Test All
+    <<: *dind-large-setup
+    timeout-minutes: 90
+    steps:
+      - <<: *checkout
+      - <<: *before-script
+      - name: Run Bazel Test All
+        id: bazel-test-all
+        uses: ./.github/actions/bazel-test-all/
+        with:
+          BAZEL_COMMAND: "test"
+          BAZEL_TARGETS: "//rs/tests/..."
```

Comment:
In gitlab CI we had `BAZEL_TARGETS: "//..."` - so we may want to keep the same for parity?

Reply:
system-tests only live under `/rs/tests` so it would be more efficient to limit bazel to that directory. I plan to do the same to the other jobs that only run system-tests but didn't want to make this PR too big.